README.md

```diff
@@ -2,8 +2,7 @@
 
 # Leaderboard App
 
->I am setting up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I have developped a first working version of the app following a wireframe, but with minimum styling just focusing on the functionality. Next I've implemented the API functionality. A user can post their favourite Game of Thrones character with their score which should be less than 1000. The data is send to the API and when the user clicks on the refresh button, the data is gotten from the API and displayed on the page. On refreshing the page, the correct lists is gotten from the API and displayed.
-
+>I am setting up a JavaScript project for the Leaderboard list app, using Webpack and ES6 features, notably modules. I have developed a first working version of the app following a wireframe, but with minimum styles just focusing on the functionality. Next, I've implemented the API functionality. A user can post their favorite Game of Thrones character with their score, which should be less than 1000. The data is then sent to the API and when the user clicks on the refresh button, the data is gotten from the API and displayed on the page. On refreshing the page, the correct list is gotten from the API and displayed. For the final milestone, I added some styles to improve the look and feel of the application.
 
 ## Built With
 
```

Comment:
- [ ] The demo link on your ReadMe is **not updated** and therefore invalid. As this is the final touch to this project and may potentially feature on your portfolio, you are therefore **required** to have professional ReadMe for this project. Kindly host this project of GitHub Pages, Netlify, or the like, and provide the appropriate demo link on your ReadMe. Also, change the title _Webpack_ to **Live Demo** or similar.

**Your Demo Link:**
![image](https://user-images.githubusercontent.com/12745474/137473205-23c4e845-4a1f-42ae-8edd-83c5813413ce.png)

Reply:
Deploying the project was not part of the requirements. The link I provided is the local host link. If we were told to deployed I would have done so and provided the link.

Reply:
Hi @Kingstalux 
You may simply remove the link. But just make sure that your ReadMe is clean. However, maintaining a professional ReadMe is a requirement